packages/typescript-estree/src/useProgramFromProjectService.ts

```diff
@@ -1,43 +1,45 @@
 import { minimatch } from 'minimatch';
+import path from 'path';
 
 import { createProjectProgram } from './create-program/createProjectProgram';
 import type { ProjectServiceSettings } from './create-program/createProjectService';
-import {
-  type ASTAndDefiniteProgram,
-  ensureAbsolutePath,
-  getCanonicalFileName,
-} from './create-program/shared';
+import { type ASTAndDefiniteProgram } from './create-program/shared';
```

Comment:
todo enable [`import/consistent-type-specifier-style`](https://github.com/import-js/eslint-plugin-import/blob/main/docs/rules/consistent-type-specifier-style.md)

```suggestion
import type { ASTAndDefiniteProgram } from './create-program/shared';
```